scripts/gen_helm_repo.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# This script generate and copies helm charts within the helm tree of this repo.
+
+ISTIO_BASE=$(cd "$(dirname "$0")" ; pwd -P)/..
+export GOPATH=$(mktemp -d)
+WORK_DIR=${GOPATH}/src/istio.io
+CHART_OUTPUT_DIR=$ISTIO_BASE/charts
```

Comment:
I presume this stuff should be at the root of the web site? So https://istio.io/chars? If so, then the output needs to be put into the "public" directory, which is the root of the tree that gets published.

"public" is generated when you run "make gen".

Reply:
I added a command to the CI pipeline to copy ./charts to ./public/charts.
Does this fits you ?